ballerina/Ballerina.toml

```diff
@@ -5,6 +5,7 @@ version = "1.2.0"
 authors = ["Ballerina"]
 keywords = ["UDP", "datagram", "transport"]
 repository = "https://github.com/ballerina-platform/module-ballerina-udp"
+icon = "resources/icon.png"
```

Comment:
Lets have it at the ballerina project root.

Reply:
Moved.